README.md

```diff
@@ -1,9 +1,16 @@
-How to run tests on your computer:
-
-1) There are two kind of browsers that you can perform tests against.
-   -Chrome
-   -Edge
-   Please choose one of following and download corresponding WebDriver from internet.
-   If you decided to use Chrome put directory with WebDriver to $PATH as env variable. Here is article how to do it:
-   [Adding WebDriver to $PATH](https://www.selenium.dev/documentation/en/webdriver/driver_requirements/)
-   For Edge put webdriver.exe to src->test->java->resources and add /scr/test/java/resources/msedgedriver.exe to application-properties.yaml
+## How to run tests on your computer:
+
+1) There are two kind of browsers that you can perform tests against. <br />
+   * Chrome
+   * Edge <br />
```

Comment:
No need for <br /> double space at the end of the line will have the same result.
I suggest you try Visual Studio Code, it has great md files support, including style checker.